src/System.Globalization/tests/System.Globalization.Tests.csproj

```diff
@@ -27,14 +27,7 @@
     <Compile Include="NumberFormatInfo\NumberFormatInfoPercentNegativePattern.cs" />
     <Compile Include="NumberFormatInfo\NumberFormatInfoPercentPositivePattern.cs" />
     <Compile Include="TextInfo.cs" />
-    <Compile Include="CompareInfo\compareinfo.cs" />
-    <Compile Include="CompareInfo\CompareInfoCompare.cs" />
-    <Compile Include="CompareInfo\CompareInfoCompare2.cs" />
-    <Compile Include="CompareInfo\CompareInfoIndexOf.cs" />
-    <Compile Include="CompareInfo\CompareInfoIndexOf2.cs" />
-    <Compile Include="CompareInfo\CompareInfoIsPrefix.cs" />
-    <Compile Include="CompareInfo\CompareInfoIsSuffix.cs" />
-    <Compile Include="CompareInfo\CompareInfoLastIndexOf.cs" />
+    <Compile Include="CompareInfo\CompareInfoTests.cs" />
```

Comment:
If we are going to keep all the tests in 1 file, it doesn't make sense to have a folder that just has the 1 file in it.